packages/cli/src/commands/test.ts

```diff
@@ -27,14 +27,14 @@ function includesAnyOf(hayStack: string[], ...needles: string[]) {
   return false;
 }
 
-export default async (cmd: Command) => {
+export default async (_opts: OptionValues, cmd: Command) => {
   // all args are forwarded to jest
   let parent = cmd;
   while (parent.parent) {
     parent = parent.parent;
   }
-  const rawArgs = parent.rawArgs as string[];
-  const args = rawArgs.slice(rawArgs.indexOf('test') + 1);
+  const allArgs = parent.args as string[];
```

Comment:
I think this seems to be the right one - there's now an `args` and a `processedArgs` if i remember correctly. I started on a journey of trying to rewrite this one using the builtin features more but ... probably not worth it

Reply:
Alright, might have a look at cleaning it up then, but prolly safe to keep for now 😅